mono/profiler/proflog.h

```diff
@@ -16,6 +16,8 @@
  * version 8: added TYPE_RUNTIME and JIT helpers/trampolines
  * version 9: added MONO_PROFILER_CODE_BUFFER_EXCEPTION_HANDLING
  * version 10: added TYPE_COVERAGE
+ * version 11: added thread ID to TYPE_SAMPLE_HIT
+ * version 12: added more load/unload events
```

Comment:
Shouldn't it mention sample hit sorting and that the header tid is now usable for sample hits?